pkg/aws/aws.go

```diff
@@ -0,0 +1,15 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+)
+
+type Price struct {
+	unit string
+	rate string
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
field `rate` is unused (unused)